### DISCUSSION TITLE
git large file storage billing

### DISCUSSION THREAD

**Discussion by jonbgamble** [community/community#142695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The chess website lichess.org would like to use Github to version larger assets alongside the relevant source code. We are an open source non-profit with hundreds of contributors, all of which may need access to these large files. But we would drain our coffers buying data packs each month just to enable speculative contributors.

This is not a rant against a trillion dollar corporation bleeding a non profit for every megabyte possible. We have other options. But when you're charging for download bandwidth, pre-paid data packs with 1 GB free is a bad look in 2024. 

There was a post 18 months ago describing a [more lenient billing model](https://github.com/orgs/community/discussions/61362). Was that idea scuttled?

**Comment by thomassiard** [comment]
Hi there!

It sounds like your organization has a genuine need to manage larger assets efficiently and affordably. GitHub's standard options for handling large files can indeed be limiting for projects with many contributors, especially for open-source and non-profit projects like lichess.org.

You might want to explore a few alternative solutions:

GitHub’s Free Bandwidth for Open-Source Projects: If you haven’t already, reaching out to GitHub Support directly could be beneficial. They sometimes offer flexibility for bandwidth limits or other resources for open-source and non-profit organizations, especially for widely-used projects.

Git LFS Alternatives: For storing larger files, you could consider using other file storage solutions (such as AWS S3, Google Cloud Storage, or even IPFS if you’re looking for decentralized options). These can integrate with GitHub and provide scalable storage that may reduce your costs.

Sponsor Options: There are initiatives within GitHub’s sponsorships program that can sometimes assist with costs related to infrastructure for popular open-source projects. It may be worth exploring any eligibility options.

Follow Up on Billing Model Changes: Regarding the more lenient billing model you mentioned, if that’s something GitHub previously considered, it could be helpful to post on the [GitHub Community Forum](https://github.community/) or reach out directly to GitHub for clarification on any future changes to Git LFS billing for open-source projects.

**Reply by ebndev** [reply]
Hi @thomassiard, thanks for participating in GitHub Discussions!

Please note: we’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  

You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Please feel encouraged to continue engaging in Discussions as your authentic self though. We appreciate you being a part of the Community 🌟

**Comment by jonbgamble** [comment]
is this not the github community forum?
